csr/csr_test.go

```diff
@@ -65,9 +65,10 @@ func TestVerifyCSR(t *testing.T) {
 	signedReqWithEmailAddress := new(x509.CertificateRequest)
 	*signedReqWithEmailAddress = *signedReq
 	signedReqWithEmailAddress.EmailAddresses = []string{"foo@bar.com"}
-	signedReqWithIPAddress := new(x509.CertificateRequest)
-	*signedReqWithIPAddress = *signedReq
-	signedReqWithIPAddress.IPAddresses = []net.IP{net.IPv4(1, 2, 3, 4)}
+	signedReqWithURI := new(x509.CertificateRequest)
```

Comment:
Adding this URI case is good, but I think we should keep `signedReqWithIPAddress` around and change its test to expect success.

Or I guess maybe change the test name / comment to indicate that it only tests error paths.